values.yaml

```diff
@@ -351,12 +356,12 @@ postgres:
       # PostgreSQL using Helm. If false, no policies will be created for the
       # built-in database.
       enable: true
+    # postgres.default.annotations -- Apply annotations to the default postgres service.
+    # https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+    annotations: {}
 
 # services -- Kubernetes Service configuration that applies to Coder services.
 services:
-  # services.annotations -- A KV mapping of annotations. See:
-  # https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
-  annotations: {}
```

Comment:
We need to keep this for backwards compatibility and merge with the current values. @jawnsy might know how to merge values together and then toYaml it in the template